samples/cpp/benchmark_app/utils.cpp

```diff
@@ -690,12 +690,38 @@ void dump_config(const std::string& filename, const std::map<std::string, ov::An
         throw std::runtime_error("Error: Can't open config file : " + filename);
     for (auto device_it = config.begin(); device_it != config.end(); ++device_it) {
         fs << plugin_to_opencv_format(device_it->first) << "{:";
+        bool contain_device_properties = false;
+        std::map<std::string, std::map<std::string, std::string>> device_properties;
         std::stringstream strm;
         for (auto param_it = device_it->second.begin(); param_it != device_it->second.end(); ++param_it) {
-            strm << param_it->first;
             param_it->second.print(strm);
+            auto property_string = strm.str();
+            if (property_string.find(" ") != std::string::npos) {
```

Comment:
Do we need it?

Reply:
You're right. we can use a simple way to check if it is ov::device::properties. Thanks.